### PR TITLE
Added Aditya Oberai's profile details to data.json

### PR DESCRIPTION
**Pull request by adityaoberai**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -521,4 +521,10 @@
     "imageUrl": "https://user-images.githubusercontent.com/72686780/106394124-fa89d480-6420-11eb-837b-fbf179a7697d.png",
     "issueNumber": 286
   }
```
Comment (AllanRegush) [inline data.json:523]: I think we need a comma here
```suggestion
  },
```

**Review by AllanRegush — COMMENTED** [review]
Thank You :+1: Looking Good. Just a minor adjustment :nerd_face:

**Comment by adityaoberai** [comment]
Thanks @AllanRegush 
Will make the edit 😄

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you :smile:

**Comment by AllanRegush** [comment]
Oh it looks like there is a conflict. Could you resolve this? if you need any help let me know :nerd_face:

**Comment by adityaoberai** [comment]
Resolved the conflict 😄

**Review by AllanRegush — APPROVED** [review]
Looks Good :+1: Thank you :smile:

**Pull request merged**